jobs/JGLOBAL_WAVE_PREP

```diff
@@ -66,20 +66,20 @@ export COMIN=${COMIN:-${ROTDIR}/${CDUMP}.${PDY}/${cyc}/$COMPONENT}
 export COMOUT=${COMOUT:-${ROTDIR}/${CDUMP}.${PDY}/${cyc}/$COMPONENT}
 [[ ! -d $COMOUT ]] && mkdir -m 775 -p $COMOUT
 
-if [ $RUN_ENVIR = "nco" ]; then
-  export COMIN_WAV_ICE=${COMIN_WAV_ICE:-$(compath.py ${envir}/obsproc/${obsproc_ver})}/${CDUMP}.${PDY}/${cyc}/atmos
-  export COMIN_WAV_RTOFS=${COMIN_WAV_RTOFS:-$(compath.py ${envir}/${WAVECUR_DID}/${rtofs_ver})}
-else
-  if [ ! -d $DMPDIR/${WAVECUR_DID}.${RPDY} ]; then export RPDY=`$NDATE -24 ${PDY}00 | cut -c1-8`; fi
-  if [ ! -L $ROTDIR/${WAVECUR_DID}.${RPDY} ]; then # Check if symlink already exists in ROTDIR
-    $NLN $DMPDIR/${WAVECUR_DID}.${RPDY} $ROTDIR/${WAVECUR_DID}.${RPDY}
-  fi
-  if [ ! -L $ROTDIR/${CDUMP}.${PDY}/${cyc}/atmos/${WAVICEFILE} ]; then # Check if symlink already exists in ROTDIR
-    $NLN $DMPDIR/$CDUMP.${PDY}/$cyc/${WAVICEFILE} $ROTDIR/$CDUMP.${PDY}/$cyc/atmos/${WAVICEFILE}
-  fi
-  export COMIN_WAV_ICE=${COMIN_WAV_ICE:-$ROTDIR/$RUN.$PDY/$cyc/atmos}
-  export COMIN_WAV_RTOFS=${COMIN_WAV_RTOFS:-$ROTDIR}
-fi
+export COMIN_WAV_ICE=${COMIN_WAV_ICE:-$(compath.py ${envir}/obsproc/${obsproc_ver})}/${CDUMP}.${PDY}/${cyc}/atmos
+export COMIN_WAV_RTOFS=${COMIN_WAV_RTOFS:-$(compath.py ${envir}/${WAVECUR_DID}/${rtofs_ver})}
```

Comment:
The code on the left is for execution when NCO runs this.
Is this no longer NCO specific?
What about on R&D machines?

Does using the two lines above from 69, 70 remove the need for the commented code below on R&D machines?
Please confirm.

Reply:
@JessicaMeixner-NOAA is in control of this JJob, I will take recommendation from her. The reason for this change is the original logic unconditionally make slink to file using global dump directory structure without first checking the availability of the source file.

Reply:
@JessicaMeixner-NOAA is unavailable and otherwise occupied with other more pressing issues.
Please highlight specifically what the issue is and how it is impacting WCOSS2 while keeping in mind that this also should work on other machines.
If there is a need for another conditional, please illustrate the need.
If there is a failure encountered with the current logic, please indicate it in a gist for e.g.

Reply:
The availability of the source file is handled in the script, for example If the current day's RTOFS is not available, the script will use the RTOFS from the day before: 
https://github.com/NOAA-EMC/global-workflow/blob/operations/scripts/exgfs_wave_prep.sh#L647-L674 

If that RTOFS file is not there, WW3 prep will error out: 
https://github.com/NOAA-EMC/global-workflow/blob/operations/scripts/exgfs_wave_prep.sh#L711-L721 

I share in the concern expressed by @aerorahul  that the changes you made here will break this for R&D machines and I'm concerned this would also not work on WCOSS2 for EMC or NCO without the commented out sections of the code.

Reply:
please revert this change. It is already working on wcoss2 and has been extensively tested

Reply:
Reverted as requested.